### PR TITLE
feat: init project

### PR DESCRIPTION
**Pull request by lwpk110**
:tada:

### CODE REVIEW

File: bundle.Dockerfile
Comment (whg517) [inline bundle.Dockerfile:1]: this is file should be ignored!

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+!vendor/**/zz_generated.*
+
+# editor and IDE paraphernalia
+.idea
+.vscode
+*.swp
+*.swo
+*~
+
+bundle.Dockerfile
```
Comment (whg517) [inline .gitignore:28]: `bundle.Dockerfile` still track in git project

**Review by whg517 — CHANGES_REQUESTED** [review]
please modify @lwpk110

**Comment by lwpk110** [comment]
> please modify @lwpk110

pushed!

**Review by whg517 — APPROVED** [review]

**Pull request merged**